### PR TITLE
add List and Contact Models, service for add via PUT and update via PATCH

### PR DESCRIPTION
**Pull request by vakuum099**

### CODE REVIEW

File: src/contact/contact.model.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { IContact } from './contact.interfaces.js';
+
+export class Contact extends Sequelize.Model<IContact> {}
+
+Contact.init(
+  {
+    uuid: {
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4,
+      autoIncrement: true,
```
Comment (CatDidenko) [inline src/contact/contact.model.ts]: autoincrement is not needed here

File: src/contact/contact.service.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    showResult(res);
+    return addedContact201;
+  } catch (err) {
+    return internalServerError500;
+  }
+};
+
+export const updateExistedContact: WorkWithDB = async (req) => {
+  const query = updateStringHelper(req);
+  console.log(query);
```
Comment (CatDidenko) [inline src/contact/contact.service.ts]: remove console.log

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-stage-3": "^6.24.1",
+    "faker": "^5.5.3",
+    "lodash.random": "^3.2.0",
+    "lodash.times": "^4.3.2",
+    "nodemon": "^2.0.7",
+    "sequelize-cli": "^6.2.0",
+    "typescript": "^4.3.4",
+    "webpack": "^5.40.0",
+    "webpack-cli": "^4.7.2"
```
Comment (CatDidenko) [inline package.json]: You don't need webpack for building ts
Example from my project.
https://www.typescriptlang.org/docs/handbook/compiler-options.html
Снимок экрана 2021-06-26 в 00 26 02

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: src/contact/contact.service.ts (hunk 2 above)
Comment (vakuum099) [reply src/contact/contact.service.ts:23]: Changed

Comment (vakuum099) [reply src/contact/contact.service.ts]: Removed

File: src/contact/contact.model.ts (hunk 1 above)
Comment (vakuum099) [reply src/contact/contact.model.ts]: Deleted

File: package.json (hunk 3 above)
Comment (vakuum099) [reply package.json]: It works! Changed.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "microsummer",
+  "version": "1.0.0",
+  "description": "microservices via Node",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack — entry ./app/public/js/index.js - output-filename bundle.min.js - output-path ./app/public/js",
```
Comment (CatDidenko) [inline package.json]: Why you leave webpack here?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "mysql2": "^2.2.5",
+    "sequelize": "^6.6.2"
+  },
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
+    "@types/express": "^4.17.12",
+    "@types/uuid": "^8.3.0",
+    "babel-cli": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-stage-3": "^6.24.1",
```
Comment (CatDidenko) [inline package.json]: Why you need babel?

File: src/contact/contact.service.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+import { v4 as uuidv4 } from 'uuid';
+import { addedContact201, internalServerError500, updatedContact200 } from '../constants';
+import { WorkWithDB } from '../interfaces';
+import { sequelize } from '../servers';
+import { showResult, updateStringHelper } from './contact.utils';
+
+export const creatNewContact: WorkWithDB = async (req) => {
+  const { name, email } = JSON.parse(req.body.contact);
+  const query = `INSERT INTO contacts (uuid, name, email, createdAt, updatedAt) VALUES('${uuidv4()}', '${name}', '${email}', now(), now());`;
+  console.log(query);
```
Comment (CatDidenko) [inline src/contact/contact.service.ts]: remove unnecessary console.log

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (vakuum099) [reply package.json]: Removed everything unused from package.json

File: package.json (hunk 5 above)
Comment (vakuum099) [reply package.json]: Removed everything unused from package.json

File: src/contact/contact.service.ts (hunk 6 above)
Comment (vakuum099) [reply src/contact/contact.service.ts]: Removed unnecessary console.log.
Checked every file in src/ for redundant console.log's.

**Review by CatDidenko — APPROVED** [review]

**Comment by CatDidenko** [comment]
Please, resolve conflicts

**Comment by CatDidenko** [comment]
Resolve conflicts, please

**Pull request merged**